lua/entities/starfall_hologram/init.lua

```diff
@@ -25,6 +27,7 @@ function ENT:UpdateTransmitState()
 end
 
 util.AddNetworkString("starfall_hologram_clips")
+util.AddNetworkString("starfall_hologram_playercolor")
```

Comment:
I think we should use SetDTVector for the color instead of using the net lib.

Reply:
SetDTVector, SetNWVector, and SetNW2Vector all wouldn't be ideal here. The client needs know when the value changes, because in order to correctly disable player color, you have to remove the `:GetPlayerColor()` function. If you always keep the function, then you can't make models revert to their default color (which varies per model), and returning nil will just break the material proxy.

You ultimately would need a net message (or check the value every tick, obviously terrible) in order for the client to know when a serverside holo changes between having/not having player color.

Reply:
DTVars can run a function when they are changed.